pkg/lint/lintersdb/manager.go

```diff
@@ -409,6 +413,27 @@ type AnalyzerPlugin interface {
 }
 
 func (m Manager) getAnalyzerPlugin(path string) (AnalyzerPlugin, error) {
+	pathURL, err := url.Parse(path)
+	if err != nil {
+		return nil, fmt.Errorf("plugin %s has invalid path", path)
```

Comment:
if `path` is not valid path (e.g. maybe just local file system), will it fail and return here right away ?

Reply:
by valid path, what do you mean here? maybe you can refer to https://play.golang.org/p/d5poeeQM8US